backend/functions/tsconfig.json

```diff
@@ -10,5 +10,5 @@
     "target": "es2017"
   },
   "compileOnSave": true,
-  "include": ["src", "tests"]
+  "include": ["src", "tests", "../../shared/interfaces/src"]
```

Comment:
I implemented it now with a simple include statement. 
Like this, the final output directory will contain all the required js files.

<img width="284" alt="image" src="https://user-images.githubusercontent.com/1204548/190863184-6285da54-1132-4837-b003-90ea0171dc3e.png">

Feels a bit hacky since we don't have a dedicated package json for the shared directory defining the dependencies  for the shared code. But for now, it's a start.

Reply:
This is totally fine for the moment, imo. We should at some point create a separate package where we define all the types, then we won't have these ugly imports.

NPM now also supports workspaces (https://docs.npmjs.com/cli/v7/using-npm/workspaces), so I think it should pretty straightforward to set this up a local package. I'll have a look at this soon.